src/app/landpage/landpage.component.css

```diff
@@ -1,8 +1,70 @@
+*{
+  font: 100% "Comic Sans MS", cursive, sans-serif;
+}
 html, body{
-  background: url("https://cdn1.savepice.ru/uploads/2018/7/23/976e78ba9b92b97248ab74069a2820b5-full.jpg");
-  background-size: 100%;
   margin: 0;
   padding: 0;
   width: 100%;
   height: 100%;
 }
+li{
+  list-style: none;
+}
+.landpage-container{
+  background: url("https://cdn1.savepice.ru/uploads/2018/7/23/976e78ba9b92b97248ab74069a2820b5-full.jpg");
+  background-size: 100%;
+  height: 100%;
+  width: 100%;
+}
+.container-land-column{
+  color: white;
+  width: 260px;
+  background: rgba(38, 39, 39, .9);
+  float: right;
+  margin-right: 10%;
+  height: calc(100vh - 30px);
+  padding: 30px 20px 0 20px;
+  text-align: center;
+}
+.container-land-column img{
+  height: 200px;
+}
+.login input{
+  height: 30px;
+  width: 200px;
+  border-radius: 10px;
+  border: none;
+  padding-left: 10px;
+}
+.login p{
+  margin-top: 10px;
+  margin-bottom: 3px;
+  margin-right: 130px;
+}
+.login button{
+  height: 30px;
+  width: 100px;
+  border: none;
+  font-weight: bold;
+  font-size: 15px;
+  margin: 10px 0 10px 0;
+}
+#login-btn{
```

Comment:
Use Classes instead of id's for styling OR name id with "Camel case" style to visualy separate namings. Because in this case class and id look very similar and hart to maintain.